server/pkg/builtin/manifest_ja.yml

```
@@ -964,7 +964,7 @@ extensions:
       default:
         title: デフォルト
         fields:
-          buttonTitle:
+          buttonTitle: 
```

yamllint

[error] trailing spaces

(trailing-spaces)